lib/trino-hive-formats/pom.xml

```diff
@@ -211,4 +211,16 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>22</source>
+                    <target>22</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
We're not on 22 quite yet https://github.com/trinodb/trino/pull/21279

Reply:
@jklamer why do you need level 22? Any particular API or language feature?